### ISSUE TITLE
Cloudfront 403 for Nextjs 13.4 server action POST

### ISSUE THREAD

**Issue by karelnagel** [serverless-stack/sst#2948]
If using server actions in nextjs 13.4 this Cloudfront error comes up. 

```
403 ERROR
The request could not be satisfied.
This distribution is not configured to allow the HTTP request method that was used for this request. The distribution supports only cachable requests. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
Generated by cloudfront (CloudFront)
Request ID: opLHZwbJQQ44vAxAjWZ_oHvGKi_OKMKiBdR0a5ek2AZryutDxUdnrQ==
```

I was able to solve this by changing cloudfront Default(*) behaviour to allow POST methods manually and I'm wondering if I can somehow fix it in the code. Tried with two stages and both had the same issue

![image](https://github.com/serverless-stack/sst/assets/65607486/a812844f-160d-48bf-8abd-35f303078d07)

**Comment by karelnagel** [comment]
Suggested solution from AlcaponeYou#8730:

```
We can just set the default http method to allow ALL.
for now, pass in cdk.distribution.defaultBehavior.allowedMethods = AllowedMethods.ALLOW_ALL
```

**Comment by fwang** [comment]
Fixed in v2.16.3
